python/lsst/obs/base/formatters/pexConfig.py

```diff
@@ -23,54 +23,15 @@
 
 __all__ = ("PexConfigFormatter",)
 
-import os.path
-from typing import Any, Optional, Type
+from lsst.pipe.base.formatters.pexConfig import PexConfigFormatter as CurrentFormatter
 
-from lsst.daf.butler.formatters.file import FileFormatter
-from lsst.pex.config import Config
 
+class PexConfigFormatter(CurrentFormatter):
```

Comment:
Can you do just `PexConfigFormatter = CurrentFormatter` here?

Reply:
I've been burned in the past for doing things like that and messing up sphinx builds but it's highly likely this file is not included in the sphinx build anyhow. In this specific class I think I want to leave it as is so that I can have explicit documentation indicating that it should not be used (and I don't think a deprecation warning helps anyone because the only way to stop the warning would be to patch the datastore).